test/util.ts

```diff
@@ -1761,14 +1755,15 @@ describe('common/util', () => {
     // in node v4 which is why the eval call is wrapped in a try catch block.
     try {
       eval(`
+        const assert2 = require('assert');
```

Comment:
Why require assert again here?

Reply:
There's some weirdness with the combination of synthetic imports and the eval.  I get a `ReferenceError: assert is not defined` if I don't re-import it using `require`.  Not entirely sure what's happening.